src/DataGrid.tsx

```diff
@@ -336,7 +338,10 @@ function DataGrid<R, K extends keyof R, SR>({
       current.scrollTop = rowIdx * rowHeight;
     },
     selectCell,
-    commitChanges: handleCommit2
+    commitChanges() {
```

Comment:
Do we use this api ourselves? I wonder what's the use case for this.

Reply:
It was added as initially we were not committing the value on outside click. It is probably not needed now. Let's revisit it
https://github.com/adazzle/react-data-grid/pull/2102#discussion_r463855118